react/react2/react2-week1-githubuser/src/components/Header.jsx

```diff
@@ -0,0 +1,4 @@
+const Header = () => {
+  return <h1> Github user searcher</h1>;
```

Comment:
Why do this when you can just put in the h1 in the App function directly?

Reply:
Thats true.